pages/getting-started.mdx

```diff
@@ -96,11 +96,10 @@ You can connect to the Memgraph instance using the command-line interface
 mgconsole by running the following command in a new terminal:
 
 ```terminal
-# use `docker ps` to get the Memgraph container id for memgraph-mage
-docker exec -ti <container-id> mgconsole
+docker run -it memgraph/mgconsole:latest --host HOST_IP --port PORT
```

Comment:
Let's surround `--host HOST_IP` and `--port PORT` in square `[`brackets`]` to indicate they are optional.